components/apr-tooltip/AprTooltip.tsx

```diff
@@ -114,6 +114,12 @@ function AprTooltip({ data, textProps, onlySparkles, placement, aprLabel, sparkl
                             </Box>
                         );
                     })}
+                    {data.items.find((item) => item.title === 'Voting APR*') && (
```

Comment:
nit: could seperate this out to a variable

Reply:
Done!